src/locale/locales/de/messages.po

```diff
@@ -4125,15 +4125,15 @@ msgstr "Stummschaltung aufheben"
 
 #: src/components/TagMenu/index.web.tsx:90
 msgid "Unmute {truncatedTag}"
-msgstr ""
+msgstr "Stummschaltung aufheben"
```

Comment:
The `{truncatedTag}` is missing here, should it be this?

```suggestion
msgstr "Stummschaltung aufheben {truncatedTag}"
```

Or a different word order?

Reply:
```suggestion
msgstr "Stummschaltung von {truncatedTag} aufheben"
```

good catch